core/examples/obsolete/loader.py

```diff
@@ -377,7 +383,7 @@ def testChunkTable(self, chunkPrefix, chunkId, partTable):
             INNER JOIN %s AS p
             ON (c.chunkId = p.chunkId AND c.subChunkId = p.subChunkId)
             WHERE c.numRows != p.numRows""" %
-            (chunkTable, partTable))
+                            (chunkTable, partTable))
```

Comment:
Identation is strange because string is enclosed between """. I'll change nothing here as code is in obsolete.